workbench-core/authentication/src/plugins/cognitoAuthenticationPlugin.ts

```diff
@@ -89,12 +97,13 @@ export class CognitoAuthenticationPlugin implements AuthenticationPlugin {
     cognitoDomain,
     userPoolId,
     webUiClient,
-    allowedClientIds = []
+    allowedClientIds = [],
+    tokenRevocationService
```

Comment:
same here

Reply:
Confirmed that the list of client Ids is related to a new feature that was added to develop.